framework/components/ema-separator/ema-separator.twig

```diff
@@ -0,0 +1,37 @@
+{#
+  Accepts:
+    - title (string): title of the panel (default: '')
+    - icon (string): name of the ECL-icons class to be used in the heading
+    - extra_classes (string): additional classes to add to the component
+    - extra_attributes (array of { name: '', value: '' })
+#}
+
+{# Internal properties #}
+{% set _css_class = 'ema-separator' %}
+{% set _extra_attributes = '' %}
+{% set _title = title|default('') %}
+{% set _icon = icon ~ ' ecl-icon'|default('') %}
```

Comment:
icon classes will be 'ema-icon ema-icon--<icon_name>' (once we add them).
So by default _icon should contain everything but the icon name (or empty string if icon is not provided).
Icon name is provided in 'icon' parameter (easier than passing the full class name).

Even if ema icons are not ready yet, it will work directly when we add them.